config/locales/titles.en.yml

```diff
@@ -47,6 +47,18 @@ en:
       index: "Phone Calls"
     remote_phone_call_events:
       index: "Remote Phone Call Events"
+    sensors:
+      index: "Sensors"
+      new: "New Sensor"
+      create: "Create Sensor"
```

Comment:
Can remove `create` and `update` keys as they're translated from the `helpers` key